katello_certs_tools/fileutils.py

```diff
@@ -226,7 +223,8 @@ def rhn_popen(cmd, progressCallback=None, bufferSize=16384, outputLog=None):
 
     return exitcode, child_out, child_err
 
-def makedirs(path,  mode=0755, user=None, group=None):
+
+def makedirs(path,  mode='0755', user=None, group=None):
```

Comment:
`os.mkdir` (as used in line 252) does not accept string parameters (at least not in Python 3.6 and 3.7)